meta/Meta.cpp

```diff
@@ -986,37 +994,6 @@ sai_status_t Meta::meta_validate_stats(
     return SAI_STATUS_SUCCESS;
 }
 
-sai_status_t Meta::meta_validate_query_stats_capability(
```

Comment:
why this is removed ?

Reply:
This had the same MACRO checks as we do in the actual functions. If it is mandatory to keep this function alive, i can add it back.